code/modules/species/shadekin/shadekin_abilities.dm

```diff
@@ -54,8 +54,8 @@
 	if(!(ability_flags & AB_PHASE_SHIFTED))
 		log_debug("[src] attempted to shift with [watcher] visible Carbons with a  cost of [ability_cost] in a darkness level of [darkness]")
 
-	var/datum/species/shadekin/SK = species
-	if(!istype(SK))
+	//var/datum/species/shadekin/SK = species
```

Comment:
this check should happen much earlier(on of the first checks

Reply:
Moved it further up. The other abilities had it pretty high up already.